src/main/java/org/fundacionjala/sfdc/core/driver/DriverManager.java

```diff
@@ -23,7 +24,7 @@
      * Constructor, private to apply singleton pattern.
      */
     private DriverManager() {
-        PropertyConfigurator.configure("log4j.properties");
+        //PropertyConfigurator.configure("log4j2.properties");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)